text/0000-sends-accepts.md

```diff
@@ -0,0 +1,46 @@
+- Start Date: 2015-12-3
+- RFC PR: (leave this empty)
+- Refract Issue: (leave this empty)
+
+# Summary
+
+This proposes adding `sends` and `accepts` attributes to the `Category` element
+and `Transaction` element in the API Description format.
+
+# Motivation
+
+This accommodates the ability to capture what mime types an API or Transaction
+will send or accept. This will provide hints to the clients and server
+developers into what mime types are supported.
+
+# Detailed design
+
+This is the definition of these two attributes:
+
+- `sends` (array[string]) - Defines what MAY be used in the `Content-Type` header
+- `accepts` (array[string]) - Defines what MAY be used in the `Accepts` header
+
+When either of these are not present, these values are determined by those in
+the HTTP Request and HTTP Response element. These values do not dictate what
+mime types a Transaction MUST use, but rather acts as hints.
+
+These attributes should be added to `Category` and `Transaction` elements.
+
+# Drawbacks
+
+This is adding information that is only for hinting, and may not actually
```

Comment:
I should say, we should encourage users to include a transaction for each given media type.